blocks/api/registration.js

```diff
@@ -87,6 +94,24 @@ export function getUnknownTypeHandler() {
 }
 
 /**
+ * Assigns the default block slug
+ *
+ * @param {string} slug Block slug
+ */
+export function setDefaultBlockName( slug ) {
```

Comment:
Per prior note, we should probably change this from `slug` to `name` in anticipation of #974.